namespaces/Representor.md

```diff
@@ -0,0 +1,133 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Representor (Object Element)
+
+The Representor represents a Hypermedia Resource, with it's available links, transitions and attributes.
+
+### Properties
+
+- `links` - (Array Element of Link) - The links from this resource.
```

Comment:
Also I am afraid, the `Link` in this namespace is clashing with `Link` from the base spec. Sure we can namespace it but maybe it would be easier to pick another name